docs/sources/operators-guide/tools/mimirtool.md

```diff
@@ -92,6 +92,14 @@ mimirtool alertmanager load <config_file>
 mimirtool alertmanager load <config_file> <template_files>...
 ```
 
+### Validate Alertmanager configuration
```

Comment:
The `##### Example`  below is related to "Load Alertmanager configuration". I think you should:
1. Move `### Validate Alertmanager configuration` after the example below (right before `#### Delete Alertmanager configuration`)
2. Change `### Validate Alertmanager configuration` to `#### Validate Alertmanager configuration` (4 hashes) to keep formatting consistent